README.md

```diff
@@ -32,46 +32,75 @@ openSonect: function() {
         "navigationBarTitleImage": "Bank"
     };
 
+    // Configure an array of payment methods with the metadata to be displayed in Withdraw screen
     let paymentMethods = [
-                          {
-                          uniqueIdentifier: "IBAN_1",
-                          name: "My Bank",
-                          detailDescription: "Balance: 20CHF",
-                          image: "Bank"
-                          },
+        {
+        uniqueIdentifier: "IBAN_1",
+        name: "My Bank",
+        detailDescription: "Balance: 20CHF",
+        image: "Bank"
+        },
     ];
 
+    // Present the sonect SDK with the credentials obtained from your server, payment methods and a theme
     sonect.present(credentials, paymentMethods, theme,
-                   function(uniqueIdentifier, balanceCallback) {
-                   //This method should check against the bank balance and return a balance.
-                   let balance = {
-                        uniqueIdentifier: uniqueIdentifier,
-                        value: "20.00",
-                        currency: "CHF"
-                   };
-                   balanceCallback(balance);
-                   },
-                   function(uniqueIdentifier, value, currency, paymentCallback) {
-                   //This method should initiate bank payment and return a payment reference as a string.
-                   let paymentReference = {
-                        uniqueIdentifier: uniqueIdentifier,
-                        paymentReference: "PAYMENT_REFERENCE"
-                   };
-                   paymentCallback(paymentReference);
-                   },
-                   function(msg) {
-                   document
-                   .getElementById('deviceready')
-                   .querySelector('.received')
-                   .innerHTML = msg;
-                   },
-                   function(err) {
-                   document
-                   .getElementById('deviceready')
-                   .innerHTML = '<p class="event received">' + err + '</p>';
-                   })
+        // When the user selects a payment method, this callback is asked if that payment method
+        // has enough balance to process the transaction. If this is less than the picked value
+	// then the Confirm button with be greyed out. uniqueIdentifier is the identifier of the payment
```

Comment:
Start spacing issue

Reply:
This is so weird... It looks OK in the branch and in the viewer, yet here it's indented by one char 🤷‍♀️